### DISCUSSION TITLE
I'm new to roasting; Diedrich IR-7; looking for help to attach Artisian Software and Phidgets?

### DISCUSSION THREAD

**Discussion by bagelsbrews** [artisan-roaster-scope/artisan#1074] [General]
I'm not even sure where to start.  I've been manually roasting some good coffee for a few months now.  I've read everything I can find and researched in every way possible given the limited amount of time I have.  I own a coffee / bagel shop and switched over from purchasing coffee from a supplier / roaster I've used for about 3 years.  They roasted really good coffee, but I felt I needed to invest in our our coffees and price was a big factor.  Anyway, so, now I'm roasting and feel pretty comfortable making what myself and others consider to be good roasts, but I think I can be better with good software.  Problem is I don't know where to start?

Please help

Thank you,

Michael Smith 
Bagels & Brews, LLC.

**Comment by poundy** [comment]
Did you buy the roaster with the data logging option? 
https://artisan-scope.org/machines/diedrich/
The machine is supported if you did...

**Comment by bagelsbrews** [comment]
I have a 1995 Diedrich IR-7.  I don't have anything on that machine related to data logging, but I want to get it set up but not sure; actually I have no Idea of where to start.

Thank you

Michael S. Smith
Bagels & Brews, LLC.
________________________________

**Reply by poundy** [reply]
ok, so 2nd hand not new. Cool, great to know.

You basically have to consider it as any other "no monitoring" roasters. The simplest way to add this is to look at a Phidgets setup to add. Artisan Store has the RTD set https://shop.artisan.plus/shop/en/hardware/phidgets-2xrtd-set or the TC set https://shop.artisan.plus/shop/en/hardware/phidget-2xtc-set that are intended just for this purpose. 

Then, it's basically just a matter of getting the probes mounted in an appropriate location for your roaster - this doesn't vary much between roasters, you want the BT probe to be in the bean mass in the drum, often easiest to drill and tap into the door, and you want the ET to be somewhere in the exhaust air stream (or in the drum free airspace, if you are in the "ET= environmental temp" camp, but that's often harder to place the probe after-market). Look on images of other roasters the same as yours for hints.  Then it's just a matter of installing Artisan and the phidgets drivers, connecting your PC to the Phidgets controller, and some configuration.

Does that give you enough to get started with? Any more specific questions are welcome as you get deeper in!!

**Comment by bagelsbrews** [comment]
Thank you -

I got all the parts and ready to install this in the next few days.

I have what is probably a really ignorant question... when you say mount the probe deep into the bean mass.... I'm confused here.  I can't put it in the drum as the drum rotates... so, I'm assuming you mean below the drum and where the existing thermo-couple is?

Thanks for all of your help.

Michael Smith

Michael S. Smith
Bagels & Brews, LLC.
________________________________

**Reply by poundy** [reply]
no, i mean in the bean mass. 

99% of the time, people do that by attaching the probe thru the drum face. And 100% of the time, manufacturers do that... you'll probably find one on your roaster there.

(as an aside, email replies while handy and easy to respond to, are not very useful in conversations like this, which is why I only ever use the website to reply)

**Reply by bagelsbrews** [reply]
Thanks poundy.  I've (hopefully) successfully mounted the phidgets and probes.  I had to bend one of the probes pretty far to get the cover on but let's hope I didn't ruin it.  

I'm now ready to install the roasting software.  Is there a link to the open source roasting software you know of?

Thank you.

I will share some pictures on here soon that will help others with this diedrich IR7 mount.

**Reply by bagelsbrews** [reply]
![ECD5A0F0-3BCF-409B-9B1B-6C061EB9DC12](https://user-images.githubusercontent.com/122627463/220150895-078d984a-0560-4a77-a8c7-96449deb997b.jpeg)
![BFE99415-81B1-40BA-A337-5EF62FCB6105](https://user-images.githubusercontent.com/122627463/220150910-dbd514f6-4887-4e44-9013-410d1ba7d4a2.jpeg)
![C779D640-F0C6-409D-8E9B-573EA6889EC0](https://user-images.githubusercontent.com/122627463/220150932-3a5b7b3f-bdb2-4d5a-8b97-e1609ab51206.jpeg)
![63516ED8-4FF3-4AA4-A4C4-69DF0D2BE063](https://user-images.githubusercontent.com/122627463/220150959-6ccba6d0-a872-434f-8742-1ad833026fa9.jpeg)

**Reply by bagelsbrews** [reply]
I had to bend the probes, especially the one for the bean mass.  I’m not sure if I bent them too much as I’m getting no reading on Artisan from the probes.  I choose Phidgets on the software as the machine.

**Reply by poundy** [reply]
Didn't understand this: 
> I'm now ready to install the roasting software. Is there a link to the open source roasting software you know of?
since you're on the Artisan discussion area, which is exactly one of those???

I have no experience directly with Phidgets, but the most common issue can be diagnosed with the Phidgets control panel - just make sure that only the Phidgets software OR Artisan is running as they will require exclusive access to the hardware. Then, make sure you have the Phidgets tab in Artisan setup with the correct details for your setup (IDs and correct hub configs etc).  Show us screen shots of that if you can't figure it out

**Comment by MAKOMO** [comment]
Probes can carefully be bended inside the drum (not at the tip; first 1.5-2cm where the element sits!) to prevent touching the drum paddles and still having most of the probes shield inside of the drum to avoid sucking out heat via that shield to the outside of the drum.

**Reply by jsphvrghs** [reply]
That's good to know ... I was planning on attaching a longer, more sensitive probe to my machine (2kg) and the probe seemed really long in all the images I saw. Are they strong enough to withstand the weight of the beans being rotated around them? They looked quite delicate in the photos.

**Reply by poundy** [reply]
they are "consumable" in that over time they will wear out. Cheap ones from ebay last less time than more reliable "industrial" ones, but eventually you're likely to replace them. Sitting in the bean mass is not that big a deal really, but the repeated slight hammering they get can lead to internal damage and intermittent temperature display.

**Comment by bagelsbrews** [comment]
![IMG_2729](https://user-images.githubusercontent.com/122627463/219779416-4f8218ac-ee0c-4342-a899-1a6cdf0c08e2.jpeg)
![RenderedImage](https://user-images.githubusercontent.com/122627463/219779423-c8519ce0-3b7d-4fbc-b547-c2d4b985242f.JPEG)
![IMG_2976](https://user-images.githubusercontent.com/122627463/219779427-52dedc51-1f2c-42b0-9708-ee316ff0becc.JPEG)
![IMG_3594](https://user-images.githubusercontent.com/122627463/219779431-72f73fab-afc5-4e9b-8489-6af940404271.JPEG)

I purchased the Phidget set you recommended.  As far as the location to place these - I assume I place one in the bean mass area exactly where the existing one is (from the factory), and one in the exhaust.  How close or where in the exhaust should i put that probe?

Thank you.

**Reply by poundy** [reply]
it doesn't matter where in the exhaust you put it. It's trying to give you an indication of the air temp that is passing through the bean mass and exiting the drum. Many roasters have it in the bean loader / exhaust vent. Some have it in the internal air in the drum area, well out of the bean mass area (eg in some rear perforated drum roaster they may put it at the inlet area at the top of the drum).

Since I'm not familiar with these roasters, I didn't find any of your pics overly helpful in identifying existing locations I'd suggest you reuse.

**Comment by bagelsbrews** [comment]
Sorry Poundy for my ignorance with all of this... this is a learning curve for me.  What I meant was I couldn't find a link to download Artisan when I messaged you you, but have since and downloaded all the software.  I also downloaded and ran the executable phidget setup file and was able to run tests on the phidgets.  One was reporting a signal; the other did not.  I suspect I broke it when installing.  Attached are my first two graphs tracked on Artisan.  I'm having trouble with the software right now but believe I can get it down with some work.

If you don't mind, please look at these and if you have any suggestions or can point out what I'm doing wrong please let me know.

BTW, thanks for all of your help.



Michael S. Smith
Bagels & Brews, LLC.
________________________________

**Comment by CarefreeBuzzBuzz** [comment]
Read the quick start guide to give you a overview

On Tue, 21 Feb 2023 at 10:27 am bagelsbrews ***@***.***>
wrote:

> Sorry Poundy for my ignorance with all of this... this is a learning curve
> for me. What I meant was I couldn't find a link to download Artisan when I
> messaged you you, but have since and downloaded all the software. I also
> downloaded and ran the executable phidget setup file and was able to run
> tests on the phidgets. One was reporting a signal; the other did not. I
> suspect I broke it when installing. Attached are my first two graphs
> tracked on Artisan. I'm having trouble with the software right now but
> believe I can get it down with some work.
>
> If you don't mind, please look at these and if you have any suggestions or
> can point out what I'm doing wrong please let me know.
>
> BTW, thanks for all of your help.
>
>
>
> Michael S. Smith
> Bagels & Brews, LLC.
> ________________________________
> From: poundy ***@***.***>
> Sent: Monday, February 20, 2023 5:09 PM
> To: artisan-roaster-scope/artisan ***@***.***>
> Cc: Michael Smith ***@***.***>; Author ***@***.***>
> Subject: Re: [artisan-roaster-scope/artisan] I'm new to roasting; Diedrich
> IR-7; looking for help to attach Artisian Software and Phidgets?
> (Discussion #1074)
>
>
> Didn't understand this:
>
> I'm now ready to install the roasting software. Is there a link to the
> open source roasting software you know of?
> since you're on the Artisan discussion area, which is exactly one of
> those???
>
> I have no experience directly with Phidgets, but the most common issue can
> be diagnosed with the Phidgets control panel - just make sure that only the
> Phidgets software OR Artisan is running as they will require exclusive
> access to the hardware. Then, make sure you have the Phidgets tab in
> Artisan setup with the correct details for your setup (IDs and correct hub
> configs etc). Show us screen shots of that if you can't figure it out
>
> —
> Reply to this email directly, view it on GitHub<
> https://github.com/artisan-roaster-scope/artisan/discussions/1074#discussioncomment-5058043>,
> or unsubscribe<
> https://github.com/notifications/unsubscribe-auth/A5HSLB7UU5QQL7JUYLWSM43WYP2RFANCNFSM6AAAAAAT2RS4TM
> >.
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/artisan-roaster-scope/artisan/discussions/1074#discussioncomment-5058101>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIW3J4BFIGFZZKMJXSIRRG3WYP4VFANCNFSM6AAAAAAT2RS4TM>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: <artisan-roaster-scope/artisan/repo-discussions/1074/comments/5058101@
> github.com>
>
-- 

__________________
Michael Herbert, Head Roaster
*Order at www.carefreebuzzbuzz.com <http://www.carefreebuzzbuzz.com>*
Artisan RoasterScope <https://artisan-scope.org/>Team

**Reply by bagelsbrews** [reply]
Here I am again… over a year later. Phidget still working good on the pc I set it up on.  Now I’ve trying to get it working on a dell pc 8gb ram windows 10 home edition.  Having trouble.  Phidget control panel closes on me if I open artisan.  At one point it was running with artisan and I got to the point of configuring artisan with the Phidget so I could pick the Diedrich ir 7 and gas type.  Still no signal getting picked up.  Not sure what the issue is.  I do know the Phidget is send out the data as I can use it on the other maxhine

**Reply by CarefreeBuzzBuzz** [reply]
You can't have both the control panel and Artisan open at the same time.  Not sure if your issue is that simple.

**Comment by bagelsbrews** [comment]
I don’t think that is the issue, but, I’m glad you said that.  If I’m running artisan and still not getting a signal what would you suspect is wrong?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by MAKOMO** [comment]
Do you see the Phidget modules connecting correctly in the Phidgets app delivering the data you expect? If this is the case, the hardware and driver installation is fine. Remains the Artisan side of things. Most likely your Artisan configuration is wrong. You can save the Artisan settings on the PC where things go right (under menu Help), transfer the resulting `artisan-settings.aset` file to the Dell PC and load it into that instance of Artisan (again under the menu Help). This ensures that you have the same Artisan settings in both cases. If this does not work there must be Gremlins at work.

**Comment by bagelsbrews** [comment]
Thanks Marko…. downloaded the settings to new machine and that worked.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by bagelsbrews** [comment]
That’s helpful.  Thank you

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
